robotiq_force_torque_sensor/nodes/rq_sensor.cpp

```diff
@@ -177,7 +177,7 @@ int main(int argc, char **argv)
 
 	//std_msgs::String msg;
 	geometry_msgs::WrenchStamped wrenchMsg;
-	ros::param::param<std::string>("~frame_id", wrenchMsg.header.frame_id, "robotiq_force_torque_frame_id");
+	ros::param::param<std::string>("~frame_id", wrenchMsg.header.frame_id, "fts_measurement_frame");
```

Comment:
there is really no reason for a name change, since I introduced this name I felt like the "new" name is more intuitive. But you are right, if some actually already adapted this a change is bad. So I'll keep the initial frame_id and change the urdf accordingly